dot/network/messages/warp_sync_test.go

```diff
@@ -0,0 +1,30 @@
cae93e6ebd990a1d50350ac619dc36a0fe05cdd7a1654e73858586d55265f3524cc27e005be0f2376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201ca80d95b84408c14cc2c0cca1f3d2e094ebe056b5816959c887ac40b7311edf8c87e0f77a06ed398323b684e09c6d784e5333136dbae5a1d0627bf774e633408229366210b4b20adf7741fb6bc8fa669ea2f7470e1440b38969d2a47d752ffeb376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201bda1ac4056f6e6210d6b2dd8f69323cb7f2729f49ad9d42059ceecdffecebef04ce478a3cfde9bceadaf9be978443f0c3a5aea19d04b538b087cc0ca614e6b0a2bf793d9fb4251da094a2dd33bb0f68a46e60d799b58a78a4a5cc369a92ee2ec376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd620156dd1808cb6044b9d6762bbd5a0284b32e51809010300fa8b4a6f2bb3072e9e632041c6ad74ba989c3e18f3874ab4be95c2e8f51ab275d943a839c12d5b73c0a3ab4559bf186773ede8cf3ddc78a93439c8e82ead090de7d3c2fe4e75591fcf1376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd620176857f28231e3b3a8eaf85eb07b835f5f7f5d379e854debd3f325693118db758b2765fa1c4a959d41f7d0b1650f61cffe1d607eee7fb12de81777cf725cd8e073fee6e433392b821f11c4cf8fe8d17b7c08dbbc32381895955ee27448f8869e1376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201688b6d2c8640491fe4db52979dfaa2947ebded2b4ba5e020257fe29ffdb2a6c7017102fa6c9d6f0744bba28a05c34c377108bf73ee04982d6db0656208ec620c7e287589ac74a46b3a74a8ae66c3e2554857419bd00f9bfeb55957fffccd98b9376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd62012256e07d9c9f3c76356fa7946b8233591a4d9b91ed3e81b2a09b1d3ae6b69c8906a299e8245221c8efd46bc55966f1de6b2b4bd0b2b3256527d59e7b738056009e8adb538dacdd20c86c4eca9a158895cef125dfd118e883cd55454f8d59f3ed376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd62011479e687ce7b0594637250707491cbec69a909aad5dab78e00aae8c3d8a530d8de4c6a1faad6ef416aea872988d50227e35e2dfb5b73d4b3b897acaa4cf56f0ea674568468af9fef031f033166880eec9f5ddd05d797d5abf1a9b9a957c77820376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201a0e495db4830b2a4e80d045871b5f0f8541091fdb7707d5e6c8f085bc255f0a5069a0ae153d9a83f6fa3f208cd1d701fbd7e852449f026f8566c7d8826cf5b0aa99c0755eec29f2be753fb701762d1e7cc841323bac49576b9ea2e124c4b7b9f376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd62018c4f486b7b87c5735e42249c15f5ec6d7b2b50a16aa0ba3bfbe666f99537e81a6a121659a06d58eacb9594201b6644f18e54c1c6ca457a4fcfa9d15e3bfea201be0bbf54aad305682afc6dc7cd051b5dce154e7d0056e877562407872cdb1242376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201763fba0da4fef393f59f1fd9eb9a646a2a1106f404681840fa846ec56c1a3443223b8a147990c6fd3e2af11300531168be94f091931f6ac07724f3ce75a06600d3ccc285f3c648f34ca9ce476651495b9a6c4aa7607d83737bd871de56a50ed9376d697522a9a42b9c6952e771fd74be9cb19fe6dd0d41c362f753ae2686547a2fdd6201d9b7c0218e498e5930fc4ad9a8c139f774de3fda36ad408008c3957f1e822070ceda5ca810ef3fb7fda2c48576051107235edba92a3be39b38b9ac9aba8c2305f12e7ddae48873b7dcd614b8ccc8ea2e5ec000f3cc398f20801cf9047bc909b20001" //nolint:lll
```

Comment:
nit: move to a fixture?

Reply:
Looks like we do in [`dot/sync/testdata/westend_blocks.yaml`](https://github.com/ChainSafe/gossamer/blob/a46ad52411cb95dad9e706c8c725f00ff940f36d/dot/sync/testdata/westend_blocks.yaml).